<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWidget</class>
 <widget class="QWidget" name="AudioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grpState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Audio State Info</string>
     </property>
     <layout class="QVBoxLayout" name="grpStateLayout"/>
    </widget>
   </item>
  </layout>
  <action name="actionImportWaves">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-add.png</normaloff>:/icons16x16/generic-add.png</iconset>
   </property>
   <property name="text">
    <string>Import &amp;Waves</string>
   </property>
   <property name="toolTip">
    <string>Import the selected waves from the Audio Manager</string>
   </property>
  </action>
  <action name="actionRemoveWave">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/edit-delete.png</normaloff>:/icons16x16/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove Wave</string>
   </property>
   <property name="toolTip">
    <string>Remove the selected wave</string>
   </property>
   <property name="shortcut">
    <string>Del, Backspace</string>
   </property>
  </action>
  <action name="actionOrderWaveUpwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-up.png</normaloff>:/icons16x16/generic-up.png</iconset>
   </property>
   <property name="text">
    <string>Order Wave &amp;Upwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the selected wave to be ordered earlier in the sequence</string>
   </property>
  </action>
  <action name="actionOrderWaveDownwards">
   <property name="icon">
    <iconset resource="../res/HyEditor.qrc">
     <normaloff>:/icons16x16/generic-down.png</normaloff>:/icons16x16/generic-down.png</iconset>
   </property>
   <property name="text">
    <string>Order Wave &amp;Downwards</string>
   </property>
   <property name="toolTip">
    <string>Shift the selected wave to be ordered later in the sequence</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/HyEditor.qrc"/>
 </resources>
 <connections/>
</ui>
